cmd/minikube/cmd/start.go

```diff
@@ -432,6 +433,17 @@ func displayEnviron(env []string) {
 }
 
 func showKubectlInfo(kcs *kubeconfig.Settings, k8sVersion string, machineName string) error {
+	if k8sVersion == "v0.0.0" {
+		register.Reg.SetStep(register.Done)
+		out.Step(style.Ready, "Done! minikube is ready without Kubernetes!")
+		out.BoxedWithConfig(box.Config{Py: 1, Px: 4, Type: "Round", Color: "Green"}, style.Tip, "Things to try without Kubernetes ...", `- "minikube ssh" to SSH into minikube's node.
+- "minikube docker-env" to build images by pointing to the docker inside minikube
```

Comment:
Need to check for the runtime being docker, so needs a parameter with the current setting

Reply:
I was actualy aware of this I was gonna make a good first issue out of this for joey